### DISCUSSION TITLE
Missing themes.php File

### DISCUSSION THREAD

**Discussion by revolutionnaire** [tailpress/tailpress#192] [Q&A] [answered]
I just installed Composer and TailPress and when I went to the Dashboard to see the theme I created using TailPress, it gave me this error: The requested resource /themes.php was not found on this server.

I did a `git st` and it didn't show any missing themes.php file.

**Comment by jeffreyvr** [comment]
Sounds like you are missing the `wp-admin/themes.php` WordPress core file. Don't really have an explanation for why it isn't there other then maybe the download of WordPress was incomplete.

Perhaps it's best to upload the WordPress core files to your project to make sure you aren't missing any more files.

**Comment by revolutionnaire** [comment]
The Themes page of the Dashboard used to work. I just checked and the file is there `wp-admin/themes.php`. I'm using it locally. I haven't pushed the changes yet.

**Comment by jeffreyvr** [comment]
Hm, not sure what's happening here. Haven't had this issue myself.

Are there any plugins active? Have you tried activating another theme to see if that resolves the error? And just to be sure, is any theme active?

**Comment by revolutionnaire** [comment]
I can't go to the Themes page. It gives me the error. 

I have the Maintenance Mode plugin (but it used to work after I installed it) and the standard WordPress plugins that come out of the box.

I just checked the plugins and the Plugins page doesn't work either. It gives me this error: The requested resource /plugins.php was not found on this server. But, the `wp-admin/plugins.php` file is there. I just checked.

I'm using WordPress 6.2.2. Does that make a difference?

**Comment by jeffreyvr** [comment]
Which install method of TailPress did you use?

WP version should not make a difference.

**Comment by revolutionnaire** [comment]
Ah, yes! I installed Composer using Homebrew and I followed the instructions on the repository's page (the one on the top). I also didn't install WordPress as well when I did `tailpress new`. I just created a new theme.

**Comment by revolutionnaire** [comment, marked as answer]
It finally worked! I turned off the Maintenance Mode plugin and it worked. Then, I turned it on again and the theme is still working. I think you just have to turn it off and on after creating a new theme using TailPress. I guess this wasn't a known issue.

**Comment by revolutionnaire** [comment]
I tried replicating the steps I took in a new WordPress installation locally and I wasn't able to replicate the error. I don't know what caused it.
